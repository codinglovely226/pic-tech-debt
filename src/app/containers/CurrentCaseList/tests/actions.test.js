import {
  updateCaseListSearchCallerLocation,
  updateCaseListSearchCallerOrganisation,
  updateCaseListSearchCentreName,
  updateCaseListSearchFilter,
  updateCaseListSearchName,
  updateCaseListSearchPostcode,
  updateCaseListSearchSex,
  updateCaseListSearchTime,
} from '../actions';

describe('Current Case List action creators', () => {
  it('updateCaseListSearchCallerLocation', () => {
    expect(updateCaseListSearchCallerLocation('Home')).toMatchSnapshot();
  });
  it('updateCaseListSearchCallerOrganisation', () => {
    expect(updateCaseListSearchCallerOrganisation('Hospital')).toMatchSnapshot();
  });
  it('updateCaseListSearchCentreName', () => {
    expect(updateCaseListSearchCentreName('Panadol')).toMatchSnapshot();
  });
  it('updateCaseListSearchFilter', () => {
    expect(updateCaseListSearchFilter('open')).toMatchSnapshot();
  });
  it('updateCaseListSearchName', () => {
    expect(updateCaseListSearchName('Smith')).toMatchSnapshot();
  });
  it('updateCaseListSearchSex', () => {
    expect(updateCaseListSearchSex('male')).toMatchSnapshot();
  });
  it('updateCaseListSearchPostcode', () => {
    expect(updateCaseListSearchPostcode(6000)).toMatchSnapshot();
  });
  it('updateCaseListSearchTime', () => {
    expect(updateCaseListSearchTime('2018')).toMatchSnapshot();
  });
});
