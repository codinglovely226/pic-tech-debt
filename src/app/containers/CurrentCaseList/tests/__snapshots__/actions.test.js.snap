exports[`Current Case List action creators updateCaseListSearchCallerLocation 1`] = `
Object {
  "callerLocation": "Home",
  "type": "pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_CALLER_LOCATION",
}
`;

exports[`Current Case List action creators updateCaseListSearchCallerOrganisation 1`] = `
Object {
  "callerOrganisation": "Hospital",
  "type": "pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_CALLER_ORGANISATION",
}
`;

exports[`Current Case List action creators updateCaseListSearchCentreName 1`] = `
Object {
  "centreName": "Panadol",
  "type": "pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_CENTRE_NAME",
}
`;

exports[`Current Case List action creators updateCaseListSearchFilter 1`] = `
Object {
  "filter": "open",
  "type": "pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_FILTER",
}
`;

exports[`Current Case List action creators updateCaseListSearchName 1`] = `
Object {
  "name": "Smith",
  "type": "pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_NAME",
}
`;

exports[`Current Case List action creators updateCaseListSearchPostcode 1`] = `
Object {
  "postcode": 6000,
  "type": "pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_POSTCODE",
}
`;

exports[`Current Case List action creators updateCaseListSearchSex 1`] = `
Object {
  "sex": "male",
  "type": "pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_SEX",
}
`;

exports[`Current Case List action creators updateCaseListSearchTime 1`] = `
Object {
  "time": "2018",
  "type": "pic/CurrentCaseList/UPDATE_CASE_LIST_SEARCH_TIME",
}
`;
